src/main/java/de/teamlapen/vampirism/core/ModEntities.java

```diff
@@ -45,27 +45,27 @@
+    public static final EntityType<ConvertedSheepEntity> converted_sheep = prepareEntityType("converted_sheep", EntityType.Builder.create(ConvertedSheepEntity::new, EntityClassification.CREATURE).size(0.9F, 1.3F), false);
     public static final EntityType<BasicHunterEntity> vampire_hunter = prepareEntityType("vampire_hunter", EntityType.Builder.create(BasicHunterEntity::new, VReference.HUNTER_CREATURE_TYPE).size(0.6F, 1.95F), true);
     public static final EntityType<HunterTrainerEntity> hunter_trainer = prepareEntityType("hunter_trainer", EntityType.Builder.create(HunterTrainerEntity::new, VReference.HUNTER_CREATURE_TYPE).size(0.6F, 1.95F), true);
     public static final EntityType<AdvancedHunterEntity> advanced_hunter = prepareEntityType("advanced_hunter", EntityType.Builder.create(AdvancedHunterEntity::new, VReference.HUNTER_CREATURE_TYPE).size(0.6F, 1.95F), true);
     public static final EntityType<VampireBaronEntity> vampire_baron = prepareEntityType("vampire_baron", EntityType.Builder.create(VampireBaronEntity::new, VReference.VAMPIRE_CREATURE_TYPE).size(0.6F, 1.95F), true);
-    public static final EntityType<VampireMinionSaveableEntity> vampire_minion_s = getNull();
-    public static final EntityType<DummyBittenAnimalEntity> dummy_creature = getNull();
-    public static final EntityType<ConvertedVillagerEntity> villager_converted = getNull();
-    public static final EntityType<AggressiveVillagerEntity> villager_angry = getNull();
-    public static final EntityType<CrossbowArrowEntity> crossbow_arrow = getNull();
-    public static final EntityType<AreaParticleCloudEntity> particle_cloud = getNull();
-    public static final EntityType<ThrowableItemEntity> throwable_item = getNull();
-    public static final EntityType<DraculaHalloweenEntity> special_dracula_halloween = getNull();
-    public static final EntityType<DarkBloodProjectileEntity> dark_blood_projectile = getNull();
-    public static final EntityType<SoulOrbEntity> soul_orb = getNull();
-    public static final EntityType<HunterFactionVillagerEntity> villager_hunter_faction = getNull();
-    public static final EntityType<VampireFactionVillagerEntity> villager_vampire_faction = getNull();
-    public static final EntityType<DummyHunterTrainerEntity> hunter_trainer_dummy = getNull();
+    public static final EntityType<VampireMinionSaveableEntity> vampire_minion_s = prepareEntityType("vampire_minion_s", EntityType.Builder.create(VampireMinionSaveableEntity::new, VReference.VAMPIRE_CREATURE_TYPE).size(0.5F, 1.1F), false);
+    public static final EntityType<DummyBittenAnimalEntity> dummy_creature = prepareEntityType("dummy_creature", EntityType.Builder.create(DummyBittenAnimalEntity::new, EntityClassification.CREATURE), false);
+    public static final EntityType<ConvertedVillagerEntity> villager_converted = prepareEntityType("villager_converted", EntityType.Builder.create(ConvertedVillagerEntity::new, VReference.VAMPIRE_CREATURE_TYPE).size(0.6F, 1.95F), false);
```

Comment:
Why are they assigned directly instead of via object holder?

Reply:
because i need the a few entities before they are registered -> not work with getNull() while not the object

Reply:
and for visual purpose i moved everything up

Reply:
where? for the spawn part. That can be moved to common setup?
Creating things during registry events is cleaner than doing so on class load. Also the object holder stuff makes sure everything is properly registered

Reply:
Vampire Forest need them and the spawneggs needs them too

Reply:
Alright, seems like there is no way around this.